scripts/update-integrations.mjs

```diff
@@ -199,14 +231,55 @@ Updated: ${existingIntegrations.size - deprecatedIntegrations.length} integratio
 	console.info(stats);
 }
 
-async function safeUpdateExistingIntegrations() {
+async function urlStatus (url) {
+	try {
+		await axios.head(url, {timeout: 10000});
```

Comment:
Similarly, not sure we need to add `axios` as a dependency here. We can achieve similar with a native `fetch(url, { method: 'HEAD' })`. Doesn’t throw an error or timeout, but can still check the status and could even be made to timeout with an [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) if that’s very important (speed isn’t essential to these scripts though IMO, so definitely OK to skip that if you prefer).